backend/src/test/java/com/team4/backend/controller/InternshipOfferControllerTest.java

```diff
@@ -43,7 +42,25 @@ void addAnInternshipOffer() {
                 .bodyValue(internshipOfferDTO)
                 .exchange()
                 //ASSERT
-                .expectStatus().isOk()
+                .expectStatus().isCreated()
+                .expectBody(InternshipOfferDto.class);
```

Comment:
Previous PR comment will change this

Reply:
https://github.com/0xD34DC0DE/ProjetIntegreTeam4/pull/18/commits/d2ed1c85f6be22103575d99fd05bb075e89feae9